php/Blockonomics.php

```diff
@@ -238,4 +292,267 @@ public function testSetup()
         // No errors
         return false;
     }
+
+    // Create the order url to redirect the user to during checkout
+    public function create_order_url($order_id, $order_url){
+        // Check if more than one crypto is activated
+        if (count($this->getActiveCurrencies()) > 1) {
+            $order_url = add_query_arg('select_crypto', $order_id, $order_url);
+        }else{
+            $order_url = add_query_arg('show_order', $order_id, $order_url);
+            $order_url = add_query_arg('crypto', 'btc', $order_url);
+        }
+        return $order_url;
+    }
+
+    // Check if a template is a nojs template
+    public function is_nojs_template($template_name){
+        if (strpos($template_name, 'nojs') === 0) {
+            return true;
+        }
+        return false;
+    }
+
+    // Check if the nojs setting is activated
+    public function is_nojs_active(){
+        return get_option('blockonomics_nojs', false);
+    }
+
+    // Check if a lite mode setting is activated
+    public function is_lite_mode_active(){
+        return get_option('blockonomics_lite', false);
+    }
+
+    // Adds the header to the blockonomics page
+    public function load_blockonomics_header($template_name){
+        add_action('wp_enqueue_scripts', 'bnomics_enqueue_stylesheets' );
+        // Apply nojs header changes
+        if (!$this->is_nojs_template($template_name)) {
+            add_action('wp_enqueue_scripts', 'bnomics_enqueue_scripts' );
+        }
+        // Apply lite-mode header
+        if($this->is_lite_mode_active()){
+        ?>
+          <link rel="stylesheet" type="text/css" href="<?php echo plugins_url('css/order.css', dirname(__FILE__));?>">
+        <?php
```

Comment:
Why is this necessary when bnomics_enqueue_stylesheets is already addng order.css ?

Reply:
This is necessary for lite mode as enqueued stylesheets are loaded in the wordpress header, which does not get loaded.